internal/sshdialer/ssh_dialer_test.go

```diff
@@ -506,9 +506,10 @@ func prepareSSHServer(t *testing.T) (connConfig *ConnectionConfig, cleanUp func(
 		{HostIP: connConfig.hostIPv4},
 	}
 
-	// lcow doesn't support ipv6 port bindings
+	// docker desktop doesn't support ipv6 port bindings
```

Comment:
We'll need to rebase on main for these

Reply:
Done as per https://github.com/buildpacks/pack/pull/1317